src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java

```diff
@@ -18,27 +18,49 @@
 
 import io.cdap.e2e.pages.locators.CdfStudioLocators;
 import io.cdap.e2e.utils.ConstantsUtil;
+import io.cdap.e2e.utils.ElementHelper;
 import io.cdap.e2e.utils.SeleniumDriver;
 import io.cdap.e2e.utils.SeleniumHelper;
 import org.openqa.selenium.By;
-import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.WebElement;
-
-import static io.cdap.e2e.utils.ConstantsUtil.JS_CLICK;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
- * Represents CdfStudioActions
+ * Represents Cdf Studio page Actions
  */
 public class CdfStudioActions {
+  private static final Logger logger = LoggerFactory.getLogger(CdfStudioActions.class);
   public static CdfStudioLocators cdfStudioLocators;
 
   static {
     cdfStudioLocators = SeleniumHelper.getPropertiesLocators(CdfStudioLocators.class);
   }
 
+  public static void selectDataPipelineTypeAsBatch() {
+    ElementHelper.clickOnElement(CdfStudioLocators.dataPipelineTypeDropdown);
+    ElementHelper.clickOnElementUsingJsExecutor(
+      CdfStudioLocators.getDataPipelineTypeDropdownOption(ConstantsUtil.BATCH));
+  }
+
+  public static void selectDataPipelineTypeAsRealtime() {
```

Comment:
I think we can combine `selectDataPipelineTypeAsBatch` and `selectDataPipelineTypeAsRealtime` in one method. Something like this 
`selectDataPipelineType(String type)`

Reply:
Updated.